Homework3/Program.cs

```diff
@@ -228,7 +228,9 @@ static async Task Main()
             tests(text1, "second",5);
 
             tests(text2, "last", 5);
-            
+
+            // Comment for Github
+            //////
```

Comment:
Please add each class in a separate file. Having it all in 1 place makes it to difficult to read.